Foods.yaml

```diff
@@ -6,4 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "cheese", "pie"]
```

Comment:
Removed Kale smoothies for obvious reasons.

Reply:
Spinach smoothies are the go!